plutus-core/plutus-core/src/PlutusCore/Evaluation/Machine/CostingFun/Core.hs

```diff
@@ -284,6 +309,20 @@ runCostingFunTwoArguments (CostingFun cpu mem) =
                 (runMem mem1 mem2)
 {-# INLINE runCostingFunTwoArguments #-}
 
+-- | Take an intercept, two slopes and two streams, and scale each element of
+-- the first stream by the first slope, each element of the second stream by the
+-- second slope, add the two scaled streams together, and cons the intercept to
+-- the stream afterwards.
+scaleLinearlyTwoVariables :: Intercept -> Slope -> CostStream -> Slope -> CostStream -> CostStream
+scaleLinearlyTwoVariables (Intercept intercept) (Slope slope1) stream1 (Slope slope2) stream2 =
+    addCostStream
+    (CostLast intercept)
+    (addCostStream
+     (mapCostStream (slope1 *) stream1)
+     (mapCostStream (slope2 *) stream2)
+    )
```

Comment:
I liked the previous version more than this one with inconsistent indentation of arguments, one of which is 0 spaces and the other is 1 space, none of which we normally use.

Anyways, I'm nitpicking.

Reply:
That's just what emacs did for me!  I couldn't get it to do anything better with the previous version.